packages/cubejs-api-gateway/src/SubscriptionServer.ts

```diff
@@ -157,4 +178,15 @@ export class SubscriptionServer {
   public clear() {
     this.subscriptionStore.clear();
   }
+
+  public async renewCubes(cubes) {
+    if (cubes.length === 0) {
+      return;
+    }
+
+    const subs = await this.subscriptionStore.getSubscriptionsByCubeName(cubes);
+    await Promise.all(subs.map(async subscription => {
+      await this.processMessage(subscription.connectionId, subscription.message, true);
+    }));
```

Comment:
Zoals ik je logica lees, hoeft dit niet met await, maar kan dit prima async lopen, right? Maakt je proces wat meer async.

Reply:
Ja klopt. Dit komt overeen met hoe Cube normalieter alle berichten afhandeld. Het enige verschil hierin is dat ik een andere lijst aan subscribers ophaal. 
Het is mij een beetje om het even. Vind beide wel prima :). Kan me ook wel voorstellen dat ze alles sequentieel doen om racecondities te voorkomen, omdat er mogelijk queries in deze lijst zitten die exact hetzelfde zijn. Als we dit sequentieel aflopen, kan het efficienter zijn, omdat de cache-check vooraan het process zit. Er zullen dan meer cache-hits zijn